modules/react/toast/stories/examples/RTL.tsx

```diff
@@ -10,7 +10,7 @@ export const RTL = () => {
 
   return (
     <CanvasProvider theme={{canvas: {direction: ContentDirection.RTL}}}>
-      <Toast mode="dialog">
```

Comment:
This `mode="dialog"` was removed because there's no follow-up action (i.e., no `Toast.Link`) in the RTL example, is that correct? From the component docs:

> Set the mode prop to dialog to convey the presence of a follow-up action.

Reply:
yep!